docs/machine-learning/how-to-guides/train-model-textual-ml-net.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Apply feature engineering for model training on textual data - ML.NET
+description: Learn how to apply feature engineering for model training on textual data with ML.NET
+ms.date: 11/07/2018
+ms.topic: how-to
+ms.custom: mvc
+#Customer intent: As a developer, I want to apply feature engineering for my model training on textual data with ML.NET so that I can use my model in the ML.NET processing pipeline.
+---
+# Applying feature engineering for machine learning model training on textual data with ML.NET
+
+You need to convert any non float data to `float` data types since all ML.NET `learners` expect features as a `float vector`.
+
+To learn on textual data, you need to extract text features. ML.NET has some basic text feature extraction mechanisms:
+
+- `Text normalization` (removing punctuation, diacritics, switching to lowercase etc.)
+- `Separator-based tokenization`.
+- `Stopword` removal.
+- `Ngram` and `skip-gram` extraction.
+- `TF-IDF` rescaling.
+- `Bag of words` conversion.
+
+The following example demonstrates ML.NET text feature extraction mechanisms using the 
+[Wikipedia detox dataset](https://github.com/dotnet/machinelearning/blob/master/test/data/wikipedia-detox-250-line-data.tsv):
+
+```console
+Sentiment   SentimentText
```

Comment:
I'd add a comment that this text has grammar issues intentionally. (Otherwise, I'm expecting PRs to fix it.)

Reply:
I performed a very light edit as part of the migration. These will be addressed.